sympy/series/tests/test_limits.py

```diff
@@ -220,6 +220,27 @@ def test_ceiling_requires_robust_assumptions():
     assert limit(ceiling(5 + cos(x)), x, 0, "-") == 6
 
 
+def test_frac():
+    assert limit(frac(x), x, oo) == AccumBounds(0, 1)
+    assert limit(frac(x)**x, x, oo) == AccumBounds(0, oo)  # wolfram gives (0, 1)
```

Comment:
Does gruntz work for this limit? I don't think it will but curious

Reply:
Without the block for `AccumBounds` in `sign` class , we would end up with something like the following
```
>>> gruntz(frac(x)**x, x, oo)
Traceback (most recent call last):
NotImplementedError: Result depends on the sign of sign(AccumBounds(-1, 0))
```
Hence it won't reach the heurestics method as only `PoleError/ValueError `exceptions are handled after result from `gruntz` . Hence it only makes sense to add a block to tackle `AccumBounds` in `sign` as `sign(AccumBounds(-1, 0))` does make sense if a single value can be returned ,Now we get the following

```
>>> gruntz(frac(x)**x, x, oo)
Traceback (most recent call last):
ValueError: More than one value is possible for passed argument sign(AccumBounds(-1, 0))
```
This also resolves the following
```
>>> sign(AccumBounds(-1, -0.5))
-1
>>> sign(AccumBounds(2, 3))
1
>>> sign(AccumBounds(-2, 3))
Traceback (most recent call last):
ValueError: More than one value is possible for passed argument sign(AccumBounds(-2, 3))
```